doc/simpledae/controllers.md

```diff
@@ -0,0 +1,46 @@
+# SimpleDae controllers
```

Comment:
could you pls put all of this in a single file in doc\devices\simpledae for the devices toctree